### DISCUSSION TITLE
Option for automatic updates when no clients are connected

### DISCUSSION THREAD

**Discussion by TerminatedGB** [ClassicOldSong/Apollo#113] [Ideas]
Right now I need to use another remote desktop software to remotely update Apollo. Can something like automatic updates be implemented and toggled on/off so that I don't need an additional rdp application?

**Comment by ClassicOldSong** [comment]
Installing/upgrading Apollo is a critical operation that should require manual supervision.

If it fails somehow and you happen have no physical access to your computer, you're then locked out completely.

So auto upgrading won't be implemented.

**Comment by TerminatedGB** [comment]
Is it possible to create a backup sunshine installation while an update is in progress? So if something happens with the primary installation we can revert to using a secondary installation, something like the A/B (seamless) system updates feature in android. Sorry if this wont work as I'm not an experienced programmer, but thank you so much for your work nonetheless!!

**Reply by ClassicOldSong** [reply]
I'm thinking about this too recently, currently I upgraded using RDP...
